walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -79,35 +98,38 @@
    * with O(log n) insertion time. The first phase returns a set which may have several overlapping
    * entries. Thus, the second phase, done outside of the loop, converts the raw set in O(n) time
    * into a set without overlapping entries.
+   *
+   * <p>To incorporate optional attendees, I duplicate operations (1) and (2) for optional
```

Comment:
I really like the attention to details that you add in your comments! Very clear and easy to understand.